manifests/m/Mariocube/MFAudio/11/Mariocube.MFAudio.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.MFAudio
+PackageVersion: 11
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Other%20Consoles/PlayStation/MFAudiov11.zip

```

Comment:
This is not an official mirror. The original publisher appears to be linked to on the Zophar site.

https://www.zophar.net/utilities/ps2util/mfaudio-1-1.html